setup.cfg

```diff
@@ -1,6 +1,7 @@
 [flake8]
-max_line_length = 120
-ignore = C812,WPS319,WPS305,WPS318,DAR202,WPS326,DAR301,WPS306,WPS226,WPS432
+max_line_length = 88
+ignore = WPS305,ANN101,C812,I001,I002,I003,I004,E203,E501,W503 #,WPS319,WPS318,WPS326,WPS306,WPS226,WPS432,W503
```

Comment:
Is `#` accidental?

Reply:
No, that's on purpose. Just trying to make sure I'm not ignoring things that can be easily addressed through auto-formatting.